src/app/payments/update-pending-payments.ts

```diff
@@ -179,31 +184,11 @@ const updatePendingPayment = wrapAsyncToRunInSpan({
           return settled
         }
 
-        if (status === PaymentStatus.Settled) {
-          paymentLogger.info(
-            { success: true, id: paymentHash, payment: pendingPayment },
-            "payment has been confirmed",
-          )
-
-          const revealedPreImage = lnPaymentLookup.confirmedDetails?.revealedPreImage
-          if (revealedPreImage)
-            LedgerService().updateMetadataByHash({
-              hash: paymentHash,
-              revealedPreImage,
-            })
-          if (pendingPayment.feeKnownInAdvance) return true
-
-          const { displayAmount, displayFee } = pendingPayment
-          if (displayAmount === undefined || displayFee === undefined)
-            return new UnknownLedgerError("missing display-related values in transaction")
-
-          return Wallets.reimburseFee({
-            paymentFlow,
-            journalId: pendingPayment.journalId,
-            actualFee: roundedUpFee,
-            revealedPreImage,
-          })
-        } else if (status === PaymentStatus.Failed) {
+        if (
+          status === PaymentStatus.Failed ||
+          // pendingPayment is a different version to latest payment from lnd
+          satsAmount !== toSats(paymentFlow.btcPaymentAmount.amount)
```

Comment:
this comparison should be made with invoice amount because fees can change depending on the route

Reply:
I think both sides of this _is_ the invoice amount... is it?